addons/web/controllers/export.py

```diff
@@ -562,7 +562,7 @@ def base(self, data):
         if not import_compat and groupby:
             groupby_type = [Model._fields[x.split(':')[0]].type for x in groupby]
             domain = [('id', 'in', ids)] if ids else domain
-            groups_data = Model.with_context(active_test=False).read_group(domain, ['__count'], groupby, lazy=False)
+            groups_data = Model.read_group(domain, ['__count'], groupby, lazy=False)
```

Comment:
It's mean that you cannot export archive records anymore, right ? 
If I select the filter 'Archived' and then choose some of them, it won't work anymore.

Note that, it wasn't correctly working before either because of the code in `insert_leaf` which search record without `active_test=False` in the context ...

Reply:
> It's mean that you cannot export archive records anymore, right ?
Nope, it works.